### PR TITLE
✨ write first draft of EDI section

### PR DESCRIPTION
**Pull request by FrancisBanville**

### CODE REVIEW

File: sections/4-EDI.md [hunk 1]
```diff
@@ -1 +1,11 @@
...
+
+Our list of participants is culturally diverse and respects gender parity. We mostly targeted students and early-career scientists who would benefit the most from a participation in this data-driven working group. We also invited researchers from different universities across Canada, making sure that their domains of expertise complement each other. With the aim of alleviating past and present inequalities experienced by marginalized groups and to encourage their participation in the working group, we will provide more funding to those who self-identified as being part of a marginalized community and provide a childcare service for mothers who would need it. Finally, participants who are unable to travel to Saskatchewan could attend the working group remotely if desired.
```
Comment (AshSies) [inline sections/4-EDI.md:5]: I like the idea for providing funding for those who self-identify as part of a marginalized community, but we haven't incorporated a way for participants to self-identify yet. Perhaps a pre-session survey/questionnaire could be incorporated into the pre-session activities (and could be mentioned in our work plan as well)

File: sections/4-EDI.md [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+We will discuss authorship during the first few days of the meetings prior to the writing of the manuscript. This will ensure that everyone knows their role and expected level of implication. We will use an authorship grid that specifies who will be responsible for the communication between participants, the study design, the data management, the analyses, the writing, and the final approval of the manuscript. 
+
+Our results will be shared with Indigenous communities present in Saskatchewan to have their valuable feedbacks prior to the writing of the manuscript.
```
Comment (AshSies) [inline sections/4-EDI.md:11]: Could/should we also seek consultation/approval/guidance from resident Indigenous communities prior to the working group?

**Review by AshSies — COMMENTED** [review]
Looks good! A few small comments for us to discuss (tomorrow perhaps?)

File: sections/4-EDI.md (hunk 1 above)
Comment (FrancisBanville) [reply sections/4-EDI.md:5]: That's a good point! I added a few words about the pre-session survey in the EDI section.

File: sections/4-EDI.md (hunk 2 above)
Comment (FrancisBanville) [reply sections/4-EDI.md:11]: I updated this last paragraph. Let me know what you think!

File: sections/4-EDI.md (hunk 1 above)
Comment (AshSies) [reply sections/4-EDI.md:5]: Looks great!

File: sections/4-EDI.md (hunk 2 above)
Comment (AshSies) [reply sections/4-EDI.md:11]: Looks great!

**Pull request merged**